src/translations/fr/translation.json

```diff
@@ -1,300 +1,300 @@
-    "Make bot - moderator, just type /mod streamelements in your chat": "Make bot - moderator, just type /mod streamelements in your chat",
-    "Press the \"Add new command\" button": "Press the \"Add new command\" button",
-    "Press \"Activate Command\"": "Press \"Activate Command\"",
-    "Items with the best cash back prices for leveling when buying from flea": "Items with the best cash back prices for leveling when buying from flea",
-    "Spending": "Spending",
-    "Unlocks at Loyalty Level": "Unlocks at Loyalty Level",
+    "Tarkov Tools StreamElements integration": "Intégration Tarkov Tools StreamElements",
+    "You can add command to your StreamElements bot to get price check in your twitch / youtube channel chat": "Vous pouvez ajouter une commande à votre bot StreamElements pour obtenir une vérification des prix dans le chat de votre chaîne twitch/youtube",
+    "Make bot - moderator, just type /mod streamelements in your chat": "Make bot - modérateur, tapez simplement / mod streamelements dans votre chat",
+    "Press the \"Add new command\" button": "Appuyez sur le bouton \"Ajouter une nouvelle commande\"",
+    "Press \"Activate Command\"": "Appuyez sur \"Activer la commande\"",
+    "Items with the best cash back prices for leveling when buying from flea": "Articles avec les meilleurs prix de remise en argent pour le nivellement lors de l'achat aux puces",
+    "Spending": "Dépenses",
+    "Unlocks at Loyalty Level": "Déverrouille au niveau de fidélité",
     "Prapor Items": "Prapor Items",
-    "Therapist Items": "Therapist Items",
-    "Skier Items": "Skier Items",
-    "Peacekeeper Items": "Peacekeeper Items",
-    "Mechanic Items": "Mechanic Items",
-    "Ragman Items": "Ragman Items",
-    "Jaeger Items": "Jaeger Items",
-    "Tarkov Tools Moobot integration": "Tarkov Tools Moobot integration",
-    "You can add command to your moobot to get price check in your twitch chat": "You can add command to your moobot to get price check in your twitch chat",
-    "using your twitch account": "using your twitch account",
-    "Go to Custom commands": "Go to Custom commands",
-    "Set what you want the command to be. Common is \"p\" or \"price\"": "Set what you want the command to be. Common is \"p\" or \"price\"",
-    "Press the \"Create\" button": "Press the \"Create\" button",
-    "In response choose URL Fetch - Full (plain) response": "In response choose URL Fetch - Full (plain) response",
-    "and after insert \"Command arguments\"": "and after insert \"Command arguments\"",
-    "Now press \"Save\" button": "Now press \"Save\" button",
-    "Ignore settings": "Ignore settings",
-    "Average prices": "Average prices",
-	"Booze Generator": "Booze Generator",
-	"Intelligence Center": "Intelligence Center",
-	"Lavatory": "Lavatory",
-	"Medstation": "Medstation",
-	"Nutrition Unit": "Nutrition Unit",
-	"Water Collector": "Water Collector",
-	"Workbench": "Workbench"
+    "Therapist Items": "Articles de thérapeute",
```

Comment:
In game "Therapist" is translated "La Toubib".

Can you also recheck the others, please.

Reply:
Hi I have updated that